### PR TITLE
Add BoyFun scraper

### PR DESCRIPTION
**Pull request by MortonBridges**
Add new scraper for boyfun.com and updated scrapers list accordingly.

### CODE REVIEW

File: scrapers/BoyFun.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      Date:
+        selector: //span[@class="date"]/span[@class="content"]
+        postProcess:
+          - replace:
+              - regex: (st|[nr]d|th) 
+                with: "" 
+          - parseDate: Jan 2, 2006
+      #Details: //div[@class="heading"]/following-sibling::p[1]
+      Details: 
+        selector: //div[@class="heading"]/following-sibling::text()
```
Comment (DogmaDragon) [inline scrapers/BoyFun.yml:22]: `//div[@class='content-information-description']/p/text()` also works. Any reason to not go with the more precise selector?

**Comment by DogmaDragon** [comment]
Tested, everything works.

File: scrapers/BoyFun.yml (hunk 1 above)
Comment (MortonBridges) [reply scrapers/BoyFun.yml:22]: > `//div[@class='content-information-description']/p/text()` also works. Any reason to not go with the more precise selector?

No specific reason, except that I probably missed the obvious.

File: scrapers/BoyFun.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: BoyFun
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - www.boyfun.com/video
```
[the remark this answers is not in the document]
Comment (MortonBridges) [reply scrapers/BoyFun.yml]: Thanks.  I've removed the www from the url.  Also removed an errant, unnecessary commented line.

**Pull request merged**